storage_file/models/storage_file.py

```diff
@@ -204,11 +200,11 @@ def unlink(self):
     def _clean_storage_file(self):
         # we must be sure that all the changes are into the DB since
         # we by pass the ORM
-        self.flush()
+        self.env.flush_all()
         self._cr.execute(
-            """SELECT id
+            SQL("""SELECT id
```

Comment:
Why is SQL applied? Is passing raw strings deprecated?

Reply:
Just a solution to avoid SQL injections supported by odoo. You can still continue to use query by string.

Reply:
OK, so it's your suggestion to apply `SQL` by default even if no arguments are passed here that could be used for sql injection. Fair enough.